.github/workflows/housekeeping.yml

```diff
@@ -0,0 +1,16 @@
+name: Pull-Request Houskeeping 🧹
+on: [pull_request]
+
+jobs:
+  milestone-assignment:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      pull-requests: write
+    steps:
+      - name: Assign to milestone
```

Comment:
I wonder if this should be run only on pull requests from a list of core team members--not sure it's so useful to have random contributor PRs in our milestones.

Reply:
I like the suggestion @ecraig12345 ! -> something like a declarative way how to provide list of teams or individuals would be lit 🔥

Reply:
That's a great idea! I'll work on that today.

Reply:
I'll work on having teams filtering for now, maybe in the future it makes sense to have individuals too.